data/filetype_extensions.conf

```diff
@@ -7,7 +7,7 @@ Abc=*.abc;*.abp;
 ActionScript=*.as;
 Ada=*.adb;*.ads;
 Asciidoc=*.asciidoc;*.adoc;
-ASM=*.asm;
+ASM=*.asm;*.asm51;*.a51;*.asm;*.ll;*.s03;*.s79;*.s82;*.s90;*.s;
```

Comment:
`*.asm` is duplicated.
I can't find anything using `*.ll` as an assembly extension.  Seems to be use by [Lex](https://en.wikipedia.org/wiki/Lex_%28software%29) and some obscure web stuff or something, but nothing about assembly.

`*.S` might be a nice addition, too.